src/explore-education-statistics-common/src/components/Modal.tsx

```diff
@@ -50,6 +55,18 @@ const Modal = ({
     };
   }, [fullScreen]);
 
+  const content = scrollableContent ? (
+    <div
+      className={styles.content}
+      // eslint-disable-next-line jsx-a11y/no-noninteractive-tabindex
+      tabIndex={0}
```

Comment:
The `tabIndex` here is for the scrollable area inside the modal, not the modal itself (the screenshot is of how it works currently). If we made the whole modal scrollable (is that what you're suggesting?) I don't think it would work as well as the close button wouldn't be visible until you scroll to the bottom.
![modal](https://github.com/dfe-analytical-services/explore-education-statistics/assets/81572860/077eb0f0-7ebb-40e7-86b1-7e55de0b7564)

Reply:
Fair enough, we can keep the `tabIndex` if we want to always show the Close button. However, it's still be a good idea to lock the scrolling on the main window viewport as it's a bit weird to have two scrollbars. 

I wonder if it'd be possible if we could scroll the modal content when scrolling outside the modal i.e. on the overlay. Not super necessary though.

Reply:
Scrolling - I've actually changed my mind now after testing them out on a few more screen sizes and have changed it to scroll the whole modal instead of just the content section. 

I've started switching over to Radix but it's going to be a lot of changes so I'll do that in a follow up PR instead of this one (https://dfedigital.atlassian.net/browse/EES-4567). That'll sort the scroll locking issue as well.